manifests/a/AGFEO/AGFEODashboard/2.2.0/AGFEO.AGFEODashboard.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=QUSU.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: AGFEO.AGFEODashboard
+PackageVersion: 2.2.0
+InstallerLocale: de-DE
```

Comment:
```suggestion
```

The installer doesn't seem to be in German locale. In previous manifests this was specified as `en-US`. I've seen issues with upgrades as WinGet treats this field as a requirement while upgrading, so probably best to just remove it